### PR TITLE
[ja] Update the term "Translation" to "Localization"

### PR DESCRIPTION
**Pull request by Okabe-Junya**
this pr is following up on #690

## Change List

- Update the term "Translation" to "Localization" (in the `/website/content/ja`)

Deploy preview: [link](https://deploy-preview-701--tag-app-delivery.netlify.app/ja/contribute/projects/localizations/)

### CODE REVIEW

**Comment by Okabe-Junya** [comment]
cc @hhiroshell @naonishijima @kaitoii11

**Review by naonishijima — APPROVED** [review]
LGTM

File: website/content/ja/contribute/projects/localizations/index.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 
 ## イントロダクション
 
-CNCFの素晴らしさの一端は関わっている人々の多様性とコミュニティからサポートされているからです。このコミュニティをより包括的にする一例として、可能な箇所でコンテンツの翻訳を提供することです。翻訳には当然それを作成およびメンテナンスする母語話者を必要とし、このページはTAG App Deliveryのコンテンツでのやり方を示します。
+CNCFの素晴らしさの一端は関わっている人々の多様性とコミュニティからサポートされているからです。このコミュニティをより包括的にする一例として、可能な箇所でコンテンツのローカライゼーションを提供することです。ローカライゼーションには当然それを作成およびメンテナンスする母語話者を必要とし、このページはTAG App Deliveryのコンテンツでのやり方を示します。
 
 ## ガイドライン
 
-各翻訳で異なりますが、一般的なガイドラインやノウハウは[過去の翻訳PR](https://github.com/cncf/tag-app-delivery/issues?q=label%3Atranslation+)や[Kubernetes SIG-Docsローカライゼーションドキュメント](https://kubernetes.io/ja/docs/contribute/localization/)を参考にすることをおすすめします。
+各ローカライゼーションで異なりますが、一般的なガイドラインやノウハウは[過去のローカライゼーションPR](https://github.com/cncf/tag-app-delivery/issues?q=label%3Atranslation+)や[Kubernetes SIG-Docsローカライゼーションドキュメント](https://kubernetes.io/ja/docs/contribute/localization/)を参考にすることをおすすめします。
```
Comment (hhiroshell) [inline website/content/ja/contribute/projects/localizations/index.md:12]: Thank you for your work. Let me suggest a minor fix for the sake of naturalness.

```suggestion
ローカライゼーションごとに異なりますが、一般的なガイドラインやノウハウは[過去のローカライゼーションのPR](https://github.com/cncf/tag-app-delivery/issues?q=label%3Atranslation+)や[Kubernetes SIG-Docsローカライゼーションドキュメント](https://kubernetes.io/ja/docs/contribute/localization/)を参考にすることをおすすめします。
```

**Review by hhiroshell — APPROVED** [review]
LGTM !

**Review by kaitoii11 — APPROVED** [review]
LGTM

**Comment by Okabe-Junya** [comment]
cc @thschue (as the reviewer of #690)

Since the review from sufficient localization reviewers has been completed, I would like to move forward with this PR. Thanks!

**Pull request merged**